app/src/androidTest/java/org/mozilla/fenix/ui/robots/NotificationRobot.kt

```diff
@@ -94,3 +98,14 @@ private fun mediaSystemNotificationButton(action: String) =
             .resourceId("android:id/action0")
             .descriptionContains(action)
     )
+
+private fun notificationTray() = UiScrollable(
+    UiSelector().resourceId("com.android.systemui:id/notification_stack_scroller")
```

Comment:
Same here (see above)

Reply:
It's the same ID on the following OS versions.